SpatialGDKEditorToolbar/Source/Private/SpatialGDKEditorToolbar.cpp

```diff
@@ -153,23 +153,28 @@ bool FSpatialGDKEditorToolbarModule::HandleSettingsSaved()
 	return true;
 }
 
-bool FSpatialGDKEditorToolbarModule::CanExecuteSchemaGenerator()
+bool FSpatialGDKEditorToolbarModule::CanExecuteSchemaGenerator() const
 {
 	return !bSchemaGeneratorRunning;
```

Comment:
Shouldn't we also check if SnapshotGenerator is running if we're doing this?

Reply:
Snapshot generator isn't async, so all we care about is the schema generator.